e2etests/features/eyfs_cms_signin_page_vaidation.feature

```diff
@@ -1,60 +0,0 @@
-## Example
```

Comment:
This should not have been deleted as will be used when CMS stories are picked up.

Reply:
When the stories are working, it can be brought back

Until then its clutter